department-of-veterans-affairs/caseflow
# Khalin/appeals 45867

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/APPEALS-45867)

# Description
"Create Record" button is now disabled until "Correspondence Type" is chosen.

## Acceptance Criteria

## Testing Plan

1. Steps to Reproduce: 
2. Log into Caseflow as Inbound_Ops_Team_Admin_user_S2
3. Select Queue and then Correspondence Cases
4. Select any Veteran
5. 5. Notice "Correspondence Type" says "Select..."
6. Navigate to the button of the page and view **"Create Record" button is now disabled**
